persistence/src/test/scala/hmda/persistence/processing/HmdaFileParserSpec.scala

```diff
@@ -26,17 +26,42 @@ class HmdaFileParserSpec extends ActorSpec {
   val badLines = fiCSVParseError.split("\n")
 
   "HMDA File Parser" must {
+    "persist parsed TSs" in {
+      parseTs(lines)
+      probe.send(hmdaFileParser, GetState)
+      probe.expectMsg(HmdaFileParseState(1, Nil))
+    }
+
+    "persist parsed TSs and TS parsing errors" in {
+      parseTs(badLines)
+      probe.send(hmdaFileParser, GetState)
+      probe.expectMsg(HmdaFileParseState(1, Seq(List("Timestamp is not a Long"))))
+    }
+
     "persist parsed LARs" in {
       parseLars(lines)
       probe.send(hmdaFileParser, GetState)
-      probe.expectMsg(HmdaFileParseState(3, Nil))
+      probe.expectMsg(HmdaFileParseState(4, Seq(List("Timestamp is not a Long"))))
```

Comment:
I could modify this to check for changes in the size of the persisted data... That way it would work in isolation, but without needing to restart the actors.